tomviz/python/tomviz/web.py

```diff
@@ -65,21 +58,47 @@ def web_export(executionPath, destPath, exportType, nbPhi, nbTheta):
 # -----------------------------------------------------------------------------
 
 
-def zipData(destinationPath):
-    dstFile = os.path.join(destinationPath, DATA_FILENAME)
+def bundleDataToHTML(destinationPath):
     dataDir = os.path.join(destinationPath, DATA_DIRECTORY)
+    srcHtmlPath = os.path.join(destinationPath, HTML_FILENAME)
+    dstHtmlPath = os.path.join(destinationPath, HTML_WITH_DATA_FILENAME)
+    webResources = ['<style>.webResource { display: none; }</style>']
 
     if os.path.exists(dataDir):
-        with zipfile.ZipFile(dstFile, mode='w') as zf:
-            for dirName, subdirList, fileList in os.walk(dataDir):
-                for fname in fileList:
-                    fullPath = os.path.join(dirName, fname)
-                    filePath = os.path.relpath(fullPath, dataDir)
-                    relPath = '%s/%s' % (DATA_DIRECTORY, filePath)
-                    zf.write(fullPath, arcname=relPath,
-                             compress_type=zipfile.ZIP_STORED)
-
-        shutil.rmtree(dataDir)
+        for dirName, subdirList, fileList in os.walk(dataDir):
+            for fname in fileList:
+                fullPath = os.path.join(dirName, fname)
+                filePath = os.path.relpath(fullPath, dataDir)
+                relPath = '%s/%s' % (DATA_DIRECTORY, filePath)
+                content = ''
+
+                if fname.endswith('.jpg'):
+                    with open(fullPath, 'rb') as data:
+                        dataContent = data.read()
+                        content = dataContent.encode("base64").replace('\n', '')
+                else:
+                    with open(fullPath, 'r') as data:
+                        content = data.read()
+
+                webResources.append(
+                    '<div class="webResource" data-url="%s">%s</div>'
+                    % (relPath, content))
+
+    webResources.append('<script>ready()</script></body>')
+
+    # Create new output file
+    with file(srcHtmlPath) as srcHtml:
```

Comment:
Its preferable to use open(...) here rather than calling the file constructor directly ( according to the python doc ...).